src/utils/helpers/prices.ts

```diff
@@ -1,19 +1,31 @@
-import { CollateralIdLiteral, CurrencyIdLiteral } from '@interlay/interbtc-api';
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
 import { Price, Prices } from 'common/types/util.types';
 
-const getCollateralPrice = (prices: Prices, tokenIdLiteral: CollateralIdLiteral): Price | undefined => {
+// TODO: Add BTC and IBTC to CurrencyIdLiteral in lib and remove union type
```

Comment:
I'm going to look at making the lib change now so hopefully this TODO will be handled before merging.

Reply:
This has been fixed in the UI now - we can't make a lib change, so it's been handled by creating a `ForeignAssetIdLiteral` type.